<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_row">
     <property name="text">
      <string>Row:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_column">
     <property name="text">
      <string>Column</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBox_Column"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_data">
     <property name="text">
      <string>Data:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_Row"/>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinBox_newData"/>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_UpdateCellData">
     <property name="text">
      <string>Update Cell Data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
